addons/website/static/src/js/website.ace.js

```diff
@@ -128,6 +133,7 @@ var ViewEditor = Widget.extend({
         }
         document.body.addEventListener('mouseup', stopResizing, true);
         self.$('.ace_gutter').mouseup(stopResizing).mousedown(startResizing).click(stopResizing);
+        self.$('.ace_content').mouseup(stopResizing).click(stopResizing);
```

Comment:
Yes but why is different that the preview line ".mouseup(stopResizing).mousedown(startResizing).click(stopResizing)" ?